pkg/pkgrepository/app_reconcile.go

```diff
@@ -135,6 +151,9 @@ func (a *App) reconcileFetchTemplateDeploy() exec.CmdRunResult {
 		UpdatedAt: metav1.NewTime(time.Now().UTC()),
 	}
 
+	a.appMetrics.ReconcileTimeMetrics.RegisterTemplateTime(packageRepoResourceType, a.Name(), a.Namespace(), a.appMetrics.IsFirstReconcile,
+		a.app.Status.Template.UpdatedAt.Sub(templateStartTime))
```

Comment:
@neil-hickey I think this will defeat our purpose as we want to note the time before we are calling `a.template(assetsPath)` at Line 145 and then we are taking the difference between the updatedAt time and startTime. If we do time.Now() at line 155 it will we later than UpdatedAt time we noted at line 151

Reply:
makes sense, yep!